bodybank/character_generator/models.py

```diff
@@ -107,4 +107,20 @@ class Background2Skills(models.Model):
     modifier = models.PositiveIntegerField(default=0)
 
     def __str__(self):
-        return (' '.join([str(self.background), str(self.skills), str(self.modifier)]))
\ No newline at end of file
+        return (' '.join([str(self.background), str(self.skills), str(self.modifier)]))
+
+
+class Career(models.Model):
```

Comment:
Why did you add more models? I am ok with it as long as we don't delay too much on the wizard which is a core piece of functionality

Reply:
I thought the users would like it and I was able to use the same pattern as I did for background so it was a quick add :raccoon: